library/core/src/main/java/com/google/android/exoplayer2/audio/MediaCodecAudioRenderer.java

```diff
@@ -787,6 +787,11 @@ protected MediaFormat getMediaFormat(
         mediaFormat.setFloat(MediaFormat.KEY_OPERATING_RATE, codecOperatingRate);
       }
     }
+    if (MimeTypes.AUDIO_AC4.equals(format.sampleMimeType)) {
+      // Some devices handle AC-4 raw frame by default.
```

Comment:
Hi, Andrew,
Could you write an email to me (ybai@dolby.com) so that I can reply that email to provide detailed information about your question? Thanks.